static/jade/getting-here/index.jade

```diff
@@ -139,17 +139,17 @@ block viewport
     h3 Budget ($)
 
     p.
-      Brno is called a student city for a reason. It has 6 public
+      Brno is called the student city for a reason. It has 6 public
       and 9 private universities. Due to this fact, there are many student
-      dormitories and hostels, which offer #[em ad hoc] accommodation
-      for even more than very affordable prices.
+      dormitories and hostels which offer #[em ad hoc] accommodation
+      for less than affordable prices.
 
     h4 #[a(href="http://www.hotel-palacky.cz/en/index.aspx") Hotel Palacký #[sup ***]]
 
     p.
-      This hotel is basically a part of student dormitory and it's not very far
+      This hotel is basically part of a student dormitory and it's not very far
       from the Saturday venue. It's located in a rather large student campus
-      complex at the edge of the enclosing part of the city, but you can get
+      complex at the edge of the city, but you can get
```

Comment:
Sorry! I wasn't sure and tricoder didn't help me either :smile: 
Let's say "in the same neighbourhood" instead of the whole "at the edge of..." thing.